src/App.js

```diff
@@ -1,12 +1,44 @@
 import './App.css';
+import { useState, useEffect } from 'react';
+import Face from './App/Face.js';
+import { Provider } from './Hooks/Context';
 
 function App() {
+  const [myRecipes, setMyRecipes] = useState([{name: 'Corn Fufu', image: '', lm: '12/30/20', content: '', favorite: false }])
+  const [randchange, setrandchange] = useState('.');
+  const [recipeOnView, setRecipeOnView] = useState({})
+  const [currentIndex, setCurrentIndex] = useState(0);
+  const [ activity, setActivity ] = useState([]);
+  const [ userDetails, setUserDetails ] = useState({});
+
+  useEffect(() => {
```

Comment:
I expected you to implement some `useLocalStorage` which will be cleaner. But it's fine